llvm/test/CodeGen/X86/branchfolding-landingpad-cfg.mir

```diff
@@ -0,0 +1,49 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
+# RUN: llc -mtriple=x86_64-pc-windows-msvc -run-pass=branch-folder -o - %s | FileCheck %s
+---
```

Comment:
Should attach IR in the test? Besides, without IR, how can we enable async exception?

Reply:
This test is just for branch-folder. I hacked the entire MIR just like llvm/test/CodeGen/X86/branchfolding-ehpad.mir. We don't care what IR looks like.

Reply:
See fix patch https://github.com/llvm/llvm-project/pull/77608